maybe/maybe_unix.go

```diff
@@ -0,0 +1,19 @@
+package maybe
```

Comment:
We’ll also need a package comment, but I’m assuming you would have added it after we reach agreement on the individual identifier comments…? :)

Reply:
Done.